#98 Added .DS_Store to .gitignore

lrdiaz opened the pull request:
- Laurie's comment about removing .DS_store is fixed in this pull request: #98 

- Also includes updates to formerly JSX code to TSX, for compatibility with rest of code, originally from PR #96 addressing issues #89 and #94

lrdiaz committed:
Updated JSX code to TSX

lrdiaz committed:
Created Card.tsx as reusable ListingCard template

lrdiaz committed:
Moved Card.tsx content to listing card

lrdiaz committed:
Deleted Card.tsx

lrdiaz committed:
Added .DS_store to .gitignore

zhandavidz commented on package.json:
```diff
@@ -34,5 +34,9 @@
+  },
+  "dependencies": {
+    "lucide-react": "^0.479.0",
+    "react": "^19.0.0"
```
`react` is already a dependency in `/frontend/package.json`, should not need to be a dependency here too

We should probably move `lucide-react` to `/frontend/package.json` as well

zhandavidz commented on frontend/.gitignore:
```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.sln
 *.sw?
+
+.DS_Store
```
I would move this `.DS_Store` entry in this `.gitignore` into the root `.gitignore`

zhandavidz commented on .DS_Store:
Will need to delete this .DS_Store too!

zhandavidz committed:
fix: Remove DS_Store, update gitignore and package.json

- Removed .DS_Store
- Put .DS_Store into correct .gitignore
- Updated package.json to put the dependencies in the correct package.json

Helps resolve #98 and #99

zhandavidz reviewed (approved):
These changes look good to me! Just need to resolve merge conflicts and merge!

zhandavidz commented:
LGTM! I fixed some small things and rebased it on main, I think it's ready to merge, what do yall think?

lhlee-umass commented:
Amazing, thank you Landrick!

angho8 commented:
Great job!
